site/content/docs/5.2/content/tables.md

```diff
@@ -1023,23 +1023,32 @@ Use SVG to display thumbnails or icons in your table data cell elements.
 </div>
 
 ```html
-<td>
-  <svg xmlns="http://www.w3.org/2000/svg" width="1.875rem" height="1.875rem" class="me-1" preserveAspectRatio="xMidYMid slice" focusable="false" role="img" aria-labelledby="svg1">
-    <title id="svg1">Thumbnail</title>
-    <rect width="100%" height="100%" fill="#ffd200"></rect>
-  </svg>
-  Cell text
-</td>
-<td>
-  <svg xmlns="http://www.w3.org/2000/svg" width="1.875rem" height="1.875rem" class="me-1" viewBox="0 0 30 30" role="img" aria-labelledby="svg2">
-    <title id="svg2">Document</title>
-    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#document"></use>
-  </svg>
-  Cell text
-</td>
+<div class="table-responsive">
```

Comment:
Feel like this change should be in #1710.

Reply:
I can't because this line does not exist on main branch...